versions.tf

```
@@ -3,13 +3,13 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = ">= 0.13, < 2.0"
+  required_version = "~> 1.0, < 2.0"
```

we can remove redundant `< 2.0` now..

```suggestion
  required_version = "~> 1.0"
```